include/mqtt/broker/offline_message.hpp

```diff
@@ -96,25 +96,85 @@ class offline_messages {
         catch (packet_id_exhausted_error const& e) {
             MQTT_LOG("mqtt_broker", warning)
                 << MQTT_ADD_VALUE(address, &ep)
-                << e.what();
+                << e.what()
+                << " : send the rest messages later.";
         }
         for (auto const& oflm : messages_) {
```

Comment:
Remove this loop ? message_ is empty here, correct ?

Reply:
I can't clearly define, but I mean this loop:
````C++
 for (auto const& oflm : messages_) {

Reply:
Ah, yes. 
I forgot removing it at the earlier commit. But I think that it has been removed.

Reply:
> I think that all comments are reflected to the code.
> If it works on yourr environment, I will merge the PR.

yes, i have no further comments. Please give me some time to test, i have some other duties i have to do before.

Reply:
Ok, please let me know if you finish your test.